src/app/admin/collectionCentresTable/CollectionCentresTable.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import React, { useState } from "react";
+import Table, { TableHeaders } from "@/components/Tables/Table";
+import styled from "styled-components";
+import Modal from "@/components/Modal/Modal";
+import Button from "@mui/material/Button";
+import DeleteIcon from "@mui/icons-material/Delete";
+import RefreshPageButton from "@/app/admin/RefreshPageButton";
+import { Schema } from "@/database_utils";
+import supabase from "@/supabaseClient";
+
+const DangerDialog = styled(Modal)`
+    & #deleteCollectionCentreDialog {
+        background-color: ${(props) => props.theme.error};
+    }
+`;
+
+export const OptionButtonDiv = styled.div`
+    display: flex;
+    padding-top: 1rem;
+    gap: 1rem;
+    justify-content: center;
+`;
+
+const collectionCentresTableHeaderKeysAndLabels: TableHeaders = [
+    ["name", "Name"],
+    ["acronym", "Acronym"],
+];
+
+interface Props {
+    collectionCentreData: Schema["collection_centres"][];
+}
+
+const CollectionCentresTables: React.FC<Props> = (props) => {
+    const [collectionCentreToDelete, setCollectionCentreToDelete] =
+        useState<Schema["collection_centres"]>();
+    const [refreshRequired, setRefreshRequired] = useState(false);
+
+    const collectionCentreOnDelete = (rowIndex: number): void => {
+        setCollectionCentreToDelete(props.collectionCentreData[rowIndex]); // TODO VFB-23 Change onDelete in table to return row
+    };
+
+    const onCollectionCentreDeleteConfirmation = async (): Promise<void> => {
+        const { error } = await supabase
+            .from("collection_centres")
+            .delete()
+            .eq("name", collectionCentreToDelete!.name);
+
+        // TODO VFB-25 display error message in table
```

Comment:
Oh it was outdated - originally I have a bar in VFB-25 just for displaying error message (because I have to use it in swap anyways), but I think the way Daniel makes it is that it will display somewhere else instead

Reply:
Which is yet to be done until Daniel is done with his part